unstract/connectors/src/unstract/connectors/databases/bigquery/bigquery.py

```diff
@@ -88,22 +90,71 @@ def sql_to_db_mapping(value: str) -> str:
         }
         return mapping.get(python_type, "string")
 
-    @staticmethod
-    def get_create_table_query(table: str) -> str:
+    def get_create_table_base_query(self, table: str) -> str:
+        """Function to create a base create table sql query.
+
+        Args:
+            table (str): db-connector table name
+
+        Returns:
+            str: generates a create sql base query with the constant columns
+        """
+        bigquery_table_name = str.lower(table).split(".")
+        if len(bigquery_table_name) != self.big_query_table_size:
+            raise ValueError(
+                f"Invalid table name format: '{table}'. "
+                "Please enter correct correct bigquery table in the form "
```

Comment:
```suggestion
                "Please ensure the BigQuery table is in the form of "
```
@kirtimanmishrazipstack ensure that its within line limits